### ISSUE TITLE
Tools mode: Layout issues

### ISSUE THREAD

**Issue by VortexParadox** [momentum-mod/game#1218]
**Describe the bug**
Some interface elements are misplaced, causing in some instances being unable to use the elements.
The issues are:
![image](https://user-images.githubusercontent.com/35622748/106963791-bdfef700-6706-11eb-9633-2d001c881e8d.png)
Properties list is too small  and can't be expanded.

![image](https://user-images.githubusercontent.com/35622748/106963892-df5fe300-6706-11eb-841f-60d90f9a42ee.png)
By default, the particle preview area is collapsed all the way to the right, making the rearranging of it tedious after some time.

![image](https://user-images.githubusercontent.com/35622748/106963983-028a9280-6707-11eb-9e1e-fafa8e5be0c3.png)
The engine viewport does not work as intended, with it showing a top left section of the screen instead.

![image](https://user-images.githubusercontent.com/35622748/106964655-0e2a8900-6708-11eb-8663-7ed9b50a90d7.png)
The framebuffer that's fed into the refract shader that's later on used on the previews is wrong, it shows with an offset. This was an issue before, but now with engine it can probably be taken care of.

![image](https://user-images.githubusercontent.com/35622748/106965153-d243f380-6708-11eb-960f-fbe20b2ffa62.png)
The file dialogs have their buttons don't adjust their positions when the window is transformed. They remain pinned to the top-left.

**To Reproduce**

1. Launch with `-tools`

**Expected behavior**
The behaviour should be nearly the same as the previous SDK build, where the properties list expands along with the window holding it; the engine viewport shows the world frame (tools not in focus); the framebufer used in the previews is correctly aligned with the view and at last the file dialog buttons readjust relative to the other controls

**Desktop/Branch (please complete the following information):**
 - OS: Windows 10
 - Branch: Steam, branch 0.8.8-rc Bring the Chaos

**Comment by Gocnak** [comment]
So I think the majority of these will be fixed when we drop VGUI and head over to using full Panorama. Since then, it would be using the default VGUI layouts for these tools. Added a blocked card for now because it's just pending on that work. Will close when we jump over to panorama.

**Comment by VortexParadox** [comment]
Tools on Panorama, niiice
Oh, this issue includes all instances these platform `.res` files are used, like the Hammer model browser. Hammer on Panorama as well, very niiice

**Comment by Gocnak** [comment]
Can you verify if the tools look okay in the Panorama launch option? Porting tools to Panorama is another card, and we will eventually be ditching our VGUI branch in favor of Panorama which may fix the original issue here. If the tools look/operate fine on Panorama branch (they'll still be VGUI but hopefully not broken visually/functionally), we can close this issue.

**Comment by hexaflexahexagon** [comment]
Looks like these layout issues still happen on panorama. 
![image](https://user-images.githubusercontent.com/62961532/122652144-448f0080-d10b-11eb-94aa-4196bf3a17ed.png)

**Comment by Gocnak** [comment]
That is because I am big dumbo and am compiling the panorama branch on the vgui breaking changes branch... 🤦 

Removing from 0.9.0 since we will still have VGUI then, going to add this to 0.10.0 to verify after we move to panorama fully (and get off this vgui breaking changes branch).

**Comment by hexaflexahexagon** [comment]
![image](https://user-images.githubusercontent.com/62961532/168407309-60437a0d-56e2-4405-b009-bf7dab8a30be.png)
all of the explicitly mentioned issues seem to be resolved with the 1 exception being the engine viewport. it shows as plain blue until you load a map (i did `map sj_amazon`), where it then loads for a while, shows a preview of the level with no hud, then transitions into mirroring the top-left screen. if we want to make that a separate issue then this can be closed

**Comment by Gocnak** [comment]
Yeah that can be a separate issue.
